docs/endpoints/Legacy/Two/posts.md

```diff
@@ -100,73 +96,53 @@ Example Response:
     }
 }
 ```
-## Retrieve A Specific Post
-
-```
-GET /api/v3/posts/:post_id
-```
 
-Only admins and post owners will have `tags`, `source`, and `remote_addr` returned in the response.
-
-Anonymous requests will only return accepted posts. Logged-in users can see accepted posts & any of their own pending or rejected posts. Staff can see anything!
-
-Example Response:
+## Create a Post and/or Create/Update a Signup
 
 ```
-{
-  "data": {
-    "id": 332,
-    "signup_id": 289,
-    "northstar_id": "5589c991a59dbfa93d8b45ae",
-    "media": {
-      "url": "http://localhost/storage/uploads/reportback-items/edited_332.jpeg?time=1509129880",
-      "original_image_url": "http://localhost/storage/uploads/reportback-items/289-923df5957838355206574f72d5520f0f-1509115822.jpeg?time=1509129880",
-      "text": "fe"
-    },
-    "quantity": "12",
-    "tags": [],
-    "reactions": {
-      "reacted": false,
-      "total": null
-    },
-    "status": "accepted",
-    "source": "rogue-admin",
-    "remote_addr": "",
-    "created_at": "2017-10-27T14:50:22+00:00",
-    "updated_at": "2017-10-27T14:50:22+00:00"
-  }
-}
+POST /api/v2/posts
 ```
 
-## Create a Post
-
-This will automatically create or update the corresponding signup.
-
-```
-POST /api/v3/posts
-```
+  - **northstar_id**: (string) required.
+    The northstar id of the user creating the post.
   - **campaign_id**: (int|string) required.
-    The Drupal/Contentful ID of the campaign that the user's post is associated with.
-  - **campaign_run_id**: (int)
+    The drupal node id of the campaign that the user's post is associated with. 
+  - **campaign_run_id**: (int) optional.
     The drupal campaign run node id of the campaign that the user's post is associated with.
-  - **type**: (string) required.
-    The type of post submitted e.g. photo, voter-reg, text
-  - **action**: (string) required.
-    Describes the bucket the action is tied to. A campaign could ask for multiple types of actions throughout the life of the campaign.
-  - **quantity**: (int|nullable) optional.
-    The number of reportback nouns verbed. Can be `null`.
+  - **quantity**: (int).
+    The number of reportback nouns verbed. 
   - **why_participated**: (string).
     The reason why the user participated.
-  - **text**: (string).
-    Corresponding text for the post (could be photo caption or other words). 256 max characters.
+  - **caption**: (string).
```

Comment:
Did we change this back to `caption`?

Reply:
oh shoot thanks for catching this @sbsmith86 ! I messed up and moved the v3 docs to legacy by accident - will fix now!

Reply:
@sbsmith86 actually this is correct - this is for the `v2` endpoint - I think it just got a little mixed up when I was moving files around but this should be good!